.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ executors:
   # This must match .promu.yml.
   golang:
     docker:
-      - image: cimg/go:1.21
+      - image: cimg/go:1.22.6
```

Comment:
This needs to be changed in  `.promu.yml` as well although there it should just be `1.22`. 
See here: https://github.com/prometheus/prometheus/blob/main/.promu.yml

Reply:
Commit fe96c8f1cfe52b7bb644ebed66ad3b8095555016